.gitignore

```diff
@@ -1 +1,2 @@
-vendor/
\ No newline at end of file
+vendor/
+.idea
```

Comment:
Please don't add IDE specific entries to .gitignore. You have your own global gitignore files for that.

Reply:
Okay, will remove, wait a sec.

Reply:
Fixed, will also check what's up with go.mod

Reply:
go.mod thing also seems to be fixed, although I do not quite get why this keeps happening.